Sourcery/Sourcery.swift

```diff
@@ -318,6 +320,18 @@ extension Sourcery {
 
         track("Generating code...", terminator: "")
         status = ""
+        
+        if autoLock && outputPath.exists {
+            do { try output.setImmutable(false) } catch { track("\(error)") }
```

Comment:
This works with a "happy" path when file is successfully unlocked, but will silently do nothing in case of any error. I think we should  just log with `info` scope